content/tutorials/quickstart/py-create-conditional-escrows.md

```diff
@@ -0,0 +1,625 @@
+
+### Create Conditional Escrow:
+
+<div align="center">
+<iframe width="560" height="315" src="https://www.youtube.com/embed/nWaMAr84qR0?si=LkF-vwQ3QJ4XNhJA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+</div>
+
+When you create a conditional escrow, you need to specify the `Condition` value you generated above. You must also set a cancel date and time, after which the escrow is no longer available.  
+
+To create a conditional escrow:
+
+1. Enter an **Amount** to transfer.
+2. Copy the **Operational Account** value.
+3. Paste it in the **Destination Account** field.
+4. Enter the **Escrow Condition** value.
+5. Enter the **Escrow Cancel (seconds)** value.
+6. Click **Create Escrow**.
+7. Copy and save the _Sequence Number_ of the escrow called out in the **Standby Result** field.
+
+The escrow is created on the XRP Ledger instance, reserving your requested XRP amount plus the transaction cost.
+
+When you create an escrow, capture and save the _Sequence Number_ so that you can use it to finish the escrow transaction.
+
+[![Created Escrow Transaction](img/quickstart-py-conditional-escrow-3.png)](img/quickstart-py-conditional-escrow-3.png)
+
+## Finish Conditional Escrow
+
+Any account can finish the conditional escrow any time before the _Escrow Cancel_ time. Following on the example above, you can use the _Sequence Number_ to finish the transaction once the Escrow Cancel time has passed.
+
+To finish a conditional escrow:
+
+1. Paste the sequence number in the Operational account **Sequence Number** field.
+2. Enter the **Escrow Condition** value.
+3. Enter the **Escrow Fulfillment** code for the `Condition`.
+4. Click **Finish Conditional Escrow**.
+
+The transaction completes and balances are updated for both the Standby and Operational accounts.
+
+[![Finished Escrow Transaction](img/quickstart-py-conditional-escrow-4.png)](img/quickstart-py-conditional-escrow-4.png)
+
+## Get Escrows
+
+Click **Get Escrows** for either the Standby account or the Operational account to see their current list of escrows. 
+
+## Cancel Escrow
+
+When the Escrow Cancel time passes, the escrow is no longer available to the recipient. The initiator of the escrow can reclaim the XRP, less the transaction fees. Any account can cancel an escrow once the cancel time has elapsed. Accounts that try to cancel the transaction prior to the **Escrow Cancel** time are charged the nominal transaction cost (12 drops), but the actual escrow cannot be cancelled until after the Escrow Cancel time.
+
+## Oh No! I Forgot to Save the Sequence Number!
+
+If you forget to save the sequence number, you can find it in the escrow transaction record.
+
+1. Create a new escrow as described in [Create Escrow](#create-escrow), above.
+2. Click **Get Escrows** to get the escrow information.
+3. Copy the _PreviousTxnLgrSeq_ value from the results.
```

Comment:
```suggestion
3. Copy the _PreviousTxnID_ value from the results.
```

Reply:
Not sure what to say, here. The _PreviousTxnLgrSeq_  works, the _PreviousTxnID_ doesn't work for me.